input.c

```diff
@@ -170,7 +170,7 @@ void get_input(level *lvl) {
             if (COMMANDS[i].requires_direction) {
                 dir = get_direction();
             }
-            logger("Player command: %s\n", COMMANDS[i].name);
+            //logger("Player command: %s\n", COMMANDS[i].name);
```

Comment:
Should I delete this logger() call? It's too noisy in the log for me, but it's not mine.